### ISSUE TITLE
Key preference mechanism

### ISSUE THREAD

**Issue by jeffreyssmith2nd** [EOSIO/eosjs2#39]
Add a mechanism for indicating a preference for which keys to use for signing a transaction.

**Comment by randytorres** [comment]
@jeffreyssmith2nd is this like owner vs active key?

**Comment by jeffreyssmith2nd** [comment]
Yes, except the idea being that you could have any number of arbitrary keys and choose which one you want to sign with. I believe @brandonfancher and @tbfleming had thoughts on how it should be implemented.

**Comment by tbfleming** [comment]
More like idea fragments :(
